jscomp/core/js_pass_flatten.ml

```diff
@@ -58,8 +57,8 @@ let flatten_map =
           | {statement_desc = Exp last_one ; _} :: rest_rev
             ->  
               S.block (Ext_list.rev_map_append  rest_rev 
-                [self#statement @@ S.exp (E.assign a  last_one)]
-                self#statement
+                [self.statement self (S.exp (E.assign a  last_one))]
```

Comment:
assuming we follow the design pattern, called as `self.xx self xx` 
This could be fused just as: `statement self`